README.md

```diff
@@ -2,35 +2,126 @@
 
 # Apply for postgraduate teacher training
 
-**Apply for postgraduate teacher training** service for candidates to apply to intitial teacher training courses.
+**Apply for postgraduate teacher training** is a service for candidates to apply to intitial teacher training courses.
 
-Ruby on Rails application has been set up with:
+## Installing and Running
```

Comment:
Don't capitalise words unless they're proper nouns, so this should be `Installing and running`